src/fsharp/service/service.fs

```diff
@@ -1324,13 +1324,25 @@ type TypeCheckInfo
             let underlyingTy = stripTyEqnsAndMeasureEqns g ty
             isStructTy g underlyingTy
 
+        let isValMutable (vref: ValRef) =
```

Comment:
nitpick: because this takes a `ValRef`, I suggest the function name be: `isValRefMutable`. If it just took a `Val`, then `isValMutable` would be appropriate.